src/blog/2023/03/ibmcloud-starter-removed.md

```diff
@@ -0,0 +1,70 @@
+---
+title: IBM Cloud removes Node-RED starter application
+subtitle: Get up and running with a Starter Application for Node-RED with FlowForge 
+description: IBM Cloud has recently announced that theyw ill no longer be providing their Cloud App Service Starter Kits, including the Node-RED Starter Application. Don't worry though, FlowForge has you covered.
+date: 2023-03-29
```

Comment:
Neat trick: Feel free to not use:

```suggestion
date: git created
```

Reply:
heh - I've ust suggested the opposite given 11ty's docs advise its a compute expensive way of doing it - and it seems needless to require the website to recompute this static value every single time we deploy.